pages/dashboard.vue

```diff
@@ -2,7 +2,7 @@
   <div class="page-container">
     <div class="page-contents">
       <div class="sidebar-l">
-        <div class="card page-nav">
+        <div v-if="$auth.user != null" class="card page-nav">
```

Comment:
Why not just `$auth.user`

Reply:
Just to explain what it does, having a inherent false if empty is just too cryptic to be in there